test/util/test_runners/integrations_tests_runner.py

```diff
@@ -50,42 +51,41 @@ def run_tests(self, test_modules: list):
     def start_env(self):
         """
             set up the test environment
-            1) will run a working cli to init conjurrc file
-            2) will locally create a netrc file
+            1) will locally create a netrc file
```

Comment:
order should be swapped here
1. Run the CLI to initialize the CLI to get the certificate and conjurrc files
2. Create a netrc file to provide login details

Reply:
took it thanks